Mobile-configuration.md

```diff
@@ -0,0 +1,62 @@
+# Standarize mobile configuration format
+
+## Introduction
+
+Introduce standard for system interoperability, by providing schemas that will be used to communicate between mobile sdk's and core services.
+
+## Problem Description
+
+Mobile configuration format is not standarized. It contains various fields that have no meaning for the mobile users.
+Configuration also exposes some internal fields. 
+
+```json
+{
+ "config": {
```

Comment:
I believe this issue is addressed in a previous proposal  https://github.com/aerogear/proposals/blob/master/apbs/create-secret-and-configmap-during-provision.md
Public information is stored (will be stored) in a configmap private information in a secret (the configmap is what will be turned into the configuration for the client).

Reply:
There are multiple issues here. Just pointed out the most prominent ones. 
Do you think we do not need this proposal?

Reply:
well really what we need is for the keycloak apb to implement the changes outlined in the proposal. IE put the client config stuff into a config map and other stuff into the secret. 
I am happy to have changes contributed to the CLI, just be aware we are going to be doing a lot of work around this area next sprint

Reply:
@maleck13 - I think that we should hold off with implementation till you get all your requirements. For the moment keycloak is represented as service, but there may be use cases where individual service configurations will need to have some keycloak specific data. 

Implementation for this format is trivial, but I would like to just have this PR and stage changes we are trying to implement so both SDK and Core team will have common place to innovate. Once we are happy with outcome (at some point of sprint) we could implement the changes.

Reply:
I think we can mark this as resolved, but can we add a link to the proposal here as it does seem to address the main issue outlined here